fmriprep/workflows/bold/stc.py

```diff
@@ -71,6 +71,8 @@ def init_bold_stc_wf(metadata, name='bold_stc_wf'):
                     tr='{}s'.format(metadata["RepetitionTime"]),
                     slice_timing=metadata['SliceTiming']),
         name='slice_timing_correction')
+    if 'SliceEncodingDirection' in metadata:
+        slice_timing_correction.inputs.slice_encoding_direction = metadata['SliceEncodingDirection']
```

Comment:
Maybe we could be as clear (and not hit line length limits) by just adding `slice_encoding_direction=metadata.get('SliceEncodingDirection', 'k')` to the `TShift()` call.

Reply:
OK. I have tested the previous version and this looks good
* same list order, different direction gives different results (as expected)
* flipped list, flipped direction gives same results (as expected)
* no direction vs "k" gives same results (as expected)